utf8.h

```diff
@@ -150,7 +150,7 @@ utf8_nonnull utf8_pure utf8_weak void *utf8valid(const void *str);
 // Sets out_codepoint to the next utf8 codepoint in str, and returns the address
 // of the utf8 codepoint after the current one in str.
 utf8_nonnull utf8_weak void *utf8codepoint(const void *utf8_restrict str,
-                                           long *utf8_restrict out_codepoint);
+                                           int *utf8_restrict out_codepoint);
```

Comment:
So int is not guaranteed to be 32 bits with C, whereas long is. Long will normally be sizeof(int) on my platforms I deal with (long long for 64 bits normally).